src/app/api/graphql/route.ts

```diff
@@ -115,16 +117,15 @@ const typeDefs = gql`
         author: User
         trip_id: ID
         content: String
-        name: String
         address: String
         coordinates: Coordinates
-        description?: String;
+        description: String
         name: String
     }
 
     type Coordinates {
-        latitude: Number
```

Comment:
GraphQL comes with a set of default scalar types out of the box:

    Int: A signed 32‐bit integer.
    Float: A signed double-precision floating-point value.
    String: A UTF‐8 character sequence.
    Boolean: true or false.
    ID: The ID scalar type represents a unique identifier, often used to refetch an object or as the key for a cache. The ID type is serialized in the same way as a String; however, defining it as an ID signifies that it is not intended to be human‐readable.